src/Adapters/Storage/Postgres/PostgresStorageAdapter.js

```diff
@@ -2425,9 +2425,8 @@ export class PostgresStorageAdapter implements StorageAdapter {
     return Promise.resolve();
   }
 
-  // Used for testing purposes
   async updateEstimatedCount(className: string) {
-    return this._client.none('ANALYZE $1:name', [className]);
+    return this._client.none('ANALYZE $1:name', [className]).catch(() => {});
```

Comment:
This looks like you are swallowing the error, do you think it should be logged as an error or warning? Especially since this method isn’t just used for tests anymore. This will help track down if the method stops working and can’t update the estimate in the future

Reply:
I updated it. Maybe a user doesn't have permission to run `ANALYZE`